test/module/irohad/multi_sig_transactions/gossip_propagation_strategy_test.cpp

```diff
@@ -79,7 +79,8 @@ PropagationData subscribeAndEmit(boost::optional<PropagationData> data,
   EXPECT_CALL(*pbfactory, createPeerQuery())
       .WillRepeatedly(testing::Return(boost::make_optional(
           std::shared_ptr<iroha::ametsuchi::PeerQuery>(query))));
-  GossipPropagationStrategy strategy(pbfactory, period, amount);
+  GossipPropagationStrategy strategy(
+      pbfactory, rxcpp::observe_on_event_loop(), period, amount);
```

Comment:
Event loop creates 4 threads, why not `rxcpp::observe_on_new_thread()`?

Reply:
Not always 4, but more than required here anyway
https://github.com/ReactiveX/RxCpp/blob/ce5dfc2fdd49693cd109a5f5470b73e982c33f35/Rx/v2/src/rxcpp/schedulers/rx-eventloop.hpp#L77